src/config.ts

```diff
@@ -148,3 +148,5 @@ const configLoggerLevels: { [key: string]: LoggerLevel } = {
 }
 
 export const LOGGER_LEVEL = configLoggerLevels[Config.LOGGER_LEVEL] || LoggerLevel.Debug
+
+export const SANCTIONED_COUNTRIES = ['CU', 'IR', 'KP', 'SY']
```

Comment:
We already have this data in https://github.com/valora-inc/wallet/blob/d556789f78bf105ba08f73850f95bef971f4d8b9/src/flags.ts#L16-L22

See this example for how to use it: https://github.com/valora-inc/wallet/blob/524961b631974ff3ce6ea1f8ae4db4013f7be8d7/src/verify/VerificationEducationScreen.tsx#L226-L230

Reply:
Addressed in https://github.com/valora-inc/wallet/pull/2793/commits/593053347870c8b996b4ef07e9c007711c271eda! Have some testing to do with a VPN, but I think this is close.

Reply:
@kathaypacific @jeanregisser I am not sure what the best way to test this. I can mock the users location or test the content of the screen easily enough, but I am not sure how to test the screen rendering when a users location changes.